src/sentry/usage_accountant/accountant.py

```diff
@@ -51,6 +52,7 @@ def _shutdown() -> None:
         _accountant_backend.flush()
         _accountant_backend.close()
         logger.info("Usage accountant flushed and closed.")
+        reset_backend()
```

Comment:
Why this change ?

Reply:
if people call `_shutdown` multiple times (and I think that will happen especially if we register for both atexit and celery-specific things), then I think we should not close the same backend twice.